api/db/changelog/db.changelog-264-add-user-disabled-events-table.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="dolbeew" id="changelog-264-add-user-disabled-events-table">
+    <createTable tableName="user_disabled_events">
+      <column name="id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="user_id" type="bigint">
+        <constraints nullable="false"/>
+      </column>
+      <column name="updated_by" type="text">
+        <constraints nullable="false"/>
+      </column>
+      <column name="update_time" type="datetime">
+        <constraints nullable="false"/>
+      </column>
+      <column name="admin_comment" type="varchar(255)">
+        <constraints nullable="false"/>
```

Comment:
Since this will be empty for ENABLED cases, I'd prefer nullable semantics over empty text.  What do you think?

Reply:
I don't have a strong preference, nullable works for me.